README.md

```diff
@@ -51,7 +51,7 @@ The register method supports the following parameters.
 |useFmp|Boolean|Collect FMP (first meaningful paint) data of the first screen|false|false|
 |enableSPA|Boolean|Monitor the page hashchange event and report PV, which is suitable for single page application scenarios|false|false|
 |autoTracePerf|Boolean|Support sending of performance data automatically.|false|true|
-|vue|Boolean|Support vue errors monitoring|false|undefined|
+|vue|Vue|Support vue errors monitoring|false|undefined|
```

Comment:
What does `type=Vue` means? How to set this config?

Reply:
Set it like so

```
import Vue from 'vue';

ClientMonitor.register({
    collector: 'http://127.0.0.0:8080',
    service: 'test-ui',
    pagePath: 'index.html',
    serviceVersion: 'v1.0.0',
    vue: Vue,
});
```